### PR TITLE
[Lim Feng Yue] Duke Increments

### PR DESCRIPTION
**Pull request by zwasd**

### CODE REVIEW

File: src/main/java/Chatbot.java [hunk 1]
```diff
@@ -0,0 +1,268 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
```
Comment (ncslzh) [inline src/main/java/Chatbot.java]: Public classes should have header comments to explain what it does

File: src/main/java/Chatbot.java [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+    }
+
+    /**
+     * Display what the chatbot is saying.
+     * @param message The message the chatbot is saying.
+     */
+    void say(String message) {
+        System.out.println(String.format("%s: %s\n", name, message));
+    }
+}
```
Comment (ncslzh) [inline src/main/java/Chatbot.java]: all of your methods here do not have access public/private modifier. Would be good to indicate explicitly for methods whether it is public or private although no access modifier does have it own "privacy".

File: src/main/java/TaskList.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TaskList {
+
+    private ArrayList<Item> items;
+
+    public TaskList() {
```
Comment (ncslzh) [inline src/main/java/TaskList.java]: would be good to have JavaDoc for your constructor since you have overloaded constructors

**Review by ncslzh — COMMENTED** [review]
See comments on code

**Review by kevinputera — COMMENTED** [review]
Looks great!

File: src/main/java/duke/command/AddCommand.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        }
+
+        tasks.add(task);
+        storage.save(tasks);
+        
+        String output = Message.TASK_ADDED + "\n"
+                + Message.DIVIDER + "\n"
+                + "  " + task + "\n"
+                + Message.DIVIDER + "\n"
+                + Message.showNumberOfTasks(tasks.getLength());
```
Comment (Tejas2805) [inline src/main/java/duke/command/AddCommand.java]: Good indentation

**Review by Tejas2805 — COMMENTED** [review]
Well written code! Good JavaDoc. Nice use of Command Class. The naming of Executable could be better. Overall, excellent work.

**Pull request closed without merging**